code/modules/food_and_drinks/food/snacks_pie.dm

```diff
@@ -105,6 +114,7 @@
 	..()
 	if(fey)
 		reagents.add_reagent("omnizine", 5)
+		foodtype = GRAIN | VEGETABLES
```

Comment:
Seems to be a mistake I did while resolving conflicts, thanks for pointing that out.